packages/frontend/src/components/ChooseAppAndEventSubstep/index.tsx

```diff
@@ -200,6 +209,31 @@ function ChooseAppAndEventSubstep(
             disableClearable
             disabled={editorContext.readOnly}
             options={appOptions}
+            renderOption={(optionProps, option) => (
+              <li
+                {...optionProps}
+                key={option.value.toString()}
+                style={{
+                  flexDirection: 'row',
+                }}
+              >
+                <Flex flexDir="column">
+                  <Flex gap={2} alignItems="center">
+                    <Text>{option.label}</Text>
+                    {option.description && (
+                      <Badge
+                        bgColor="interaction.muted.main.active"
+                        color="primary.600"
+                        padding="0.25rem 0.5rem"
```

Comment:
```suggestion
                        px={2}
                        py={1}
```

In general as a team coding style we prefer using chakra units as much as possible

Reply:
okayyyy thksss